packages/playwright-test/src/reporters/html.ts

```diff
@@ -294,14 +292,31 @@ class HtmlBuilder {
       }
     }
 
+    // Inline report data.
+    const indexFile = path.join(this._reportFolder, 'index.html');
+    fs.appendFileSync(indexFile, '<script>\nwindow.playwrightReportBase64 = "data:application/zip;base64,');
```

Comment:
Should we `this._dataZipFile.on('error')` and print something nice?

Reply:
Let's do it when we hit it!